README.md

```diff
@@ -138,6 +138,7 @@ vars:
 
 #### Multi-Book (Netsuite2 only)
 To include `accounting_book_id` and `accounting_book_name` columns in the end models, set the below variable to `true` in your `dbt_project.yml`. This feature is disabled by default.
+
```

Comment:
Don't forget to include the employee disable/enable documentation in the README as well since that is an upstream change.

Reply:
Brought it in!